content/providers/01-ai-sdk-providers/08-amazon-bedrock.mdx

```diff
@@ -316,6 +316,38 @@ console.log(
 // }
 ```
 
+## Reasoning
+
+Amazon Bedrock has reasoning support for the `claude-3-7-sonnet-20250219` model.
+
+You can enable it using the `reasoning_config` provider option and specifying a thinking budget in tokens (minimum: `1024`, maximum: `64000`).
+
+````ts
```

Comment:
three backticks

Reply:
lol that was definitely my editor, let me fix that